Formula/jsign.rb

```diff
@@ -0,0 +1,24 @@
+class Jsign < Formula
+  desc "Tool for signing Windows executable files, installers and scripts"
+  homepage "https://ebourg.github.io/jsign/"
+  url "https://github.com/ebourg/jsign.git", tag: "4.2", revision: "e44d2474f7773cbade092362cd4a6a63810f0c95"
+  license "Apache-2.0"
+
+  depends_on "maven" => :build
+  depends_on "openjdk@17"
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@17"].opt_prefix
+    system("mvn", "--batch-mode", "package", "--projects", "jsign-core,jsign-cli,jsign-ant,jsign", "-DskipTests",
+           "-Djdeb.skip", "-Dmaven.javadoc.skip")
+
+    libexec.install Dir["jsign/target/jsign-#{version}.jar"]
+    bin.write_jar_script libexec/"jsign-#{version}.jar", "jsign", "-Djava.net.useSystemProxies=true -Dbasename=jsign -Dlog4j2.loggerContextFactory=net.jsign.log4j.simple.SimpleLoggerContextFactory"
+    bash_completion.install "jsign/src/deb/data/usr/share/bash-completion/completions/jsign" => "jsign"
+    man1.install "jsign/src/deb/data/usr/share/man/man1/jsign.1"
+  end
+
+  test do
+    system "#{bin}/jsign", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
The intent of the test is to ensure the system integration works fine (i.e. the jar was properly installed, the launcher script generated is fine and it works with the JDK installed). The functional tests were already run upstream at release time, since it's a cross-platform tool I didn't think it was useful to run them again.

Reply:
I've added a few tests

Reply:
👍